### ISSUE TITLE
Facing issue importing the module

### ISSUE THREAD

**Issue by callmekatootie** [scribbletune/scribbletune#172]
I have executed `npm install scribbletune`.

Now, I am trying to execute the script in the README:

```js
// File is named index.js

const scribble = require('scribbletune');
const clip = scribble.clip({
    notes: scribble.scale('C4 major'),
    pattern: 'x'.repeat(7) + '_'
});

scribble.midi(clip, 'c-major.mid');
```

through the command `node index.js`. But I get the following error:

```bash
node:internal/modules/cjs/loader:361
      throw err;
      ^

Error: Cannot find module 'redacted/node_modules/scribbletune/index.js'. Please verify that the package.json has a valid "main" entry
    at tryPackage (node:internal/modules/cjs/loader:353:19)
    at Function.Module._findPath (node:internal/modules/cjs/loader:566:18)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:27)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/redacted/index.js:1:18)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32) {
  code: 'MODULE_NOT_FOUND',
  path: '/redacted/node_modules/scribbletune/package.json',
  requestPath: 'scribbletune'
}

```

**Comment by callmekatootie** [comment]
Nodejs version: 16.13.1
Scribbletune version: ^5.0.0

**Comment by walmik** [comment]
Hi @callmekatootie thank you so much for catching this. I do apologize for the delay in responding here! This issue is now fixed in v5.0.1 (i added an exports prop in the package to deal with the issue) Could you please delete node_modules directory and `npm install scribbletune` OR simply bump scribbletune to 5.0.1 in your package.json and install to see if you are able to proceed?

Please lemme know if you see any other issue!

**Comment by callmekatootie** [comment]
Thank you. That resolves it.
